src/WebUI/Components/ListComponent.scss

```diff
@@ -11,6 +11,10 @@
     .first-col-header {
         padding-left: 10px;
     }
+
+    .bolt-table-row {

```

Comment:
never use bolt-table-row, try to add a class to it and use it, as they may change this class in future